pkgs/by-name/yu/yunfaavatar/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  makeWrapper,
```

Comment:
Correct me if I'm wrong but this doesn't seem used.
```suggestion
```

Reply:
It should probably be used to inject the dependencies.

Reply:
ohh, didn't see you here you were hidden. It does do *things* on my machine, not sure it that's enough to verify it's actually working :woman_shrugging: 

```
nix run ./#yunfaavatar
[INFO] Loaded the config (/home/ces/.config/yunfaAvatar/config.conf)

[INFO] Downloading an avatar
[INFO] Avatar size is 41,889 bytes
[INFO] Converted avatar to base64

[INFO] Finished
```

Reply:
(also I can't unresolve)